service/middleware.go

```diff
@@ -48,15 +47,13 @@ type (
 
 type (
 	// custom types used to attach specific values to request contexts, to prevent collisions.
-	reqUserKey           int
 	reqTaskKey           int
 	reqProjectContextKey int
```

Comment:
do you think it makes sense to move other keys into the top level package?

Reply:
I haven't looked in to this in depth, but I think we need to keep them separate still. I tried moving all of them to the evergreen package and was not able to have my login info persist between pages